app/src/test/java/ingsw/group1/findmyphone/cryptography/RandomStringGeneratorTest.java

```diff
@@ -0,0 +1,102 @@
+package ingsw.group1.findmyphone.cryptography;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import ingsw.group1.msglibrary.SMSMessage;
+import ingsw.group1.msglibrary.SMSPeer;
+
+/**
+ * Tests the encryption and decryption with generating random text to cypher / decipher with a random generated password.
+ *
+ * @author Pardeep
+ */
+@RunWith(Parameterized.class)
+public class RandomStringGeneratorTest {
+    private final int MINIMUM_STRING_LENGTH = 1;
+    private final int MAXIMUM_STRING_LENGTH = 60;
+
+    private static final int NUM_REPEATS = 1000;
+
+    @Parameterized.Parameters()
+    public static Collection<Object[]> data() {
```

Comment:
You could change this to:
`public static Object[][] data(){
    return new Object[NUM_REPEATS][0];
}`
And it would be cleaner while still working

Reply:
Yes that's definitly much cleaner